README.md

```diff
@@ -47,37 +44,13 @@ This snippet will load `productcat.js` onto the page asynchronously, so it will
 please contact Visenze to get YOUR_APP_KEY and YOUR_CID to put in the field.
```

Comment:
still need instruction on how to set the app key, custom endpoint and various parameter. 
Please refer to Android documentation on the setup part and provide similar instruction.

Reply:
ok never mind, i saw the sample code.

Reply:
changed